### PR TITLE
Run prettier over errors reference

### PR DESCRIPTION
**Pull request by Josh-Cena**

### CODE REVIEW

File: files/en-us/web/javascript/reference/errors/illegal_character/index.md [hunk 1]
```diff
@@ -42,22 +42,22 @@ Famous examples of this are quotes, the minus or semicolon
...
 
 ```js example-bad
-“This looks like a string”;  // SyntaxError: illegal character
-                             // “ and ” are not " but look like this
+“This looks like a string”; // SyntaxError: illegal character
+// “ and ” are not " but look like it
 
-42 – 13;                     // SyntaxError: illegal character
-                             // – is not - but looks like this
+42 – 13; // SyntaxError: illegal character
```
Comment (estelle) [inline files/en-us/web/javascript/reference/errors/illegal_character/index.md:48]:
```suggestion
42 – 13; // SyntaxError: illegal character m-dash
```

File: files/en-us/web/javascript/reference/errors/not_a_codepoint/index.md [hunk 2]
```diff
@@ -42,23 +42,23 @@ Unicode codespace; that is, the range of integers from `0` to
...
-String.fromCodePoint(42);       // "*"
-String.fromCodePoint(65, 90);   // "AZ"
-String.fromCodePoint(0x404);    // "\u0404"
-String.fromCodePoint(0x2F804);  // "\uD87E\uDC04"
-String.fromCodePoint(194564);   // "\uD87E\uDC04"
-String.fromCodePoint(0x1D306, 0x61, 0x1D307) // "\uD834\uDF06a\uD834\uDF07"
+String.fromCodePoint(42); // "*"
+String.fromCodePoint(65, 90); // "AZ"
+String.fromCodePoint(0x404); // "\u0404"
+String.fromCodePoint(0x2f804); // "\uD87E\uDC04"
```
Comment (estelle) [inline files/en-us/web/javascript/reference/errors/not_a_codepoint/index.md:59]: the result is '你'

File: files/en-us/web/javascript/reference/errors/not_a_codepoint/index.md [hunk 3]
```diff
@@ -42,23 +42,23 @@ Unicode codespace; that is, the range of integers from `0` to
...
+String.fromCodePoint(194564); // "\uD87E\uDC04"
+String.fromCodePoint(0x1d306, 0x61, 0x1d307); // "\uD834\uDF06a\uD834\uDF07"
```
Comment (estelle) [inline files/en-us/web/javascript/reference/errors/not_a_codepoint/index.md:61]: "你" and "𝌆a𝌇"

Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/errors/not_a_codepoint/index.md:61]:
```suggestion
String.fromCodePoint(0x404); // 'Є' (U+0404)
String.fromCodePoint(0x2f804); // '你' (U+2F804)
String.fromCodePoint(194564); // '你'
String.fromCodePoint(0x1d306, 0x61, 0x1d307); // '𝌆a𝌇'
```

File: files/en-us/web/javascript/reference/errors/illegal_character/index.md [hunk 4]
```diff
@@ -42,22 +42,22 @@ Famous examples of this are quotes, the minus or semicolon
...
+// – is not - but looks like it
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/errors/illegal_character/index.md:49]:
```suggestion
// – (en-dash) is not - but looks like it
```

**Review by estelle — APPROVED** [review]
One question, which isn't related to your edit. Otherwise looks good.

**Comment by estelle** [comment]
You fixed it before i finished reviewing it!

**Comment by Josh-Cena** [comment]
🥷

**Comment by bsmth** [comment]
It might be good to add an exception rule to the prettier config for paths that are done like this one:

https://github.com/mdn/content/pull/24046/files#diff-b640b344ee7f3f03d2a443795a5d0708ef50e2e6e34214109ab2aad13ad6ba98R5

**Pull request merged**